thumbs/handler.py

```diff
@@ -15,38 +17,56 @@
 from django.conf import settings
 from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
 
-settings.configure(
-    THUMBNAIL_KVSTORE="thumbs.PassthruKVStore",
-)
-with open("LEAFLET_IMAGES_BUCKET_NAME") as f:
-    BUCKET_NAME = f.read().strip()
-with open("SENTRY_DSN") as f:
-    SENTRY_DSN = f.read().strip()
-
-sentry_sdk.init(
-    dsn=SENTRY_DSN,
-    # Set traces_sample_rate to 1.0 to capture 100%
-    # of transactions for tracing.
-    traces_sample_rate=0,
-    # Set profiles_sample_rate to 1.0 to profile 100%
-    # of sampled transactions.
-    # We recommend adjusting this value in production.
-    profiles_sample_rate=0,
-    integrations=[
-        AwsLambdaIntegration(),
-    ],
-)
+# Get the bucket name. Because this function is run using Lambda@Edge
+# we can't use environment variables there. We want to use these for testing
+# so we support getting the value from either.
+BUCKET_NAME = None
+if leaflet_images_bucket_name := os.environ.get("LEAFLET_IMAGES_BUCKET_NAME"):
+    BUCKET_NAME = leaflet_images_bucket_name
+else:
+    leaflets_bucket_image_path = Path("LEAFLET_IMAGES_BUCKET_NAME")
+    if leaflets_bucket_image_path.exists():
+        with open("LEAFLET_IMAGES_BUCKET_NAME") as f:
+            BUCKET_NAME = f.read().strip()
+if not BUCKET_NAME:
+    raise ValueError(
+        "LEAFLET_IMAGES_BUCKET_NAME needs to exist as an env var or file name"
+    )
 
+sentry_dsn_path = Path("SENTRY_DSN")
+if sentry_dsn_path.exists():
+    with sentry_dsn_path.open() as f:
+        SENTRY_DSN = f.read().strip()
+        sentry_sdk.init(
+            dsn=SENTRY_DSN,
+            # Set traces_sample_rate to 1.0 to capture 100%
+            # of transactions for tracing.
+            traces_sample_rate=0,
+            # Set profiles_sample_rate to 1.0 to profile 100%
+            # of sampled transactions.
+            # We recommend adjusting this value in production.
+            profiles_sample_rate=0,
+            integrations=[
+                AwsLambdaIntegration(),
+            ],
+        )
+with contextlib.suppress(RuntimeError):
```

Comment:
Not tested, but can you do `if not settings.configured:` here?
If that works, it is probably less likely to hide an error we actually do care about.
If not, lets leave it as-is.